lib/rubocop/cop/rspec/leading_subject.rb

```diff
@@ -57,21 +57,16 @@ def check_previous_nodes(node)
 
         private
 
-        def autocorrect(corrector, node)
-          first_node = find_first_offending_node(node)
+        def autocorrect(corrector, node, sibling)
           RuboCop::RSpec::Corrector::MoveNode.new(
             node, corrector, processed_source
-          ).move_before(first_node)
+          ).move_before(sibling)
         end
 
         def offending?(node)
           let?(node) || hook?(node) || example?(node)
```

Comment:
Observation:
Shouldn't `|| example_group?(node)` be added here?
I can think of:
```ruby
describe do
  describe do
    ...
  end

  subject(:i_am_offended_by_a_preceeding_example_group_where_i_am_a_full_fledged_subject) { ... }
end
```